### ISSUE TITLE
프로젝트에 할당된 모든 시퀀스, 샷 리스트가 보이는 페이지가 있었으면 좋겠습니다.

### ISSUE THREAD

**Issue by dchecheb** [digital-idea/OpenPipelineIO#565]
[RD101 요청]
- RD101은 현재 프로젝트 매니지먼트 툴로 shotgun 이용 중입니다.
- 기본적으로 프로젝트 -> 시퀀스 -> 샷 -> 태스크 -> 버전 순서대로 점점 하위로 들어가는 형식이에요
- 프로젝트 선택하면 시퀀스 리스트가 보이고 시퀀스를 선택하면 그 안의 샷 리스트가 보이고...  트리구조 느낌
- 해당 기존 작업 방식을 버리긴 힘들어 보입니다

**Comment by dchecheb** [comment]
이거 제가 저희 회사 요청에 맞게 페이지만 만들어서 pr 넣어봐도 될까요??

**Comment by khw7096** [comment]
네! 좋아요. 대신 url 페이지를 꼭 따로 만들어서 기능을 넣어주세요. :-) CSI는 digitalidea, lazypic, 프렛즐이 동시에 사용 및 테스트 하고 있어요.

**Comment by khw7096** [comment]
채윤님이 개발에 참석하게되니.. 이제 저도 PR로 코드를 올릴께요.

**Comment by khw7096** [comment]
| project list | seq list | shot list |
